### ISSUE TITLE
Live compiling doesn't work?

### ISSUE THREAD

**Issue by trnc-ck** [karthikv/nodefront#10]
hi,

i use the following command to start the compile daemon:

> nodefront compile -w -l 3001

it compiles right, but doesn't refresh chrome and safari. i'm using the latest nodefront version. 

here's the consollog:

> Compiled index.html.

thanks!

**Comment by karthikv** [comment]
What does your directory structure look like? What file are you modifying when it doesn't refresh? Remember that if the file you're modifying is in a subdirectory, you'll need to provide the `-r` or `--recursive` option to nodefront. Also, keep in mind that the `-w` option is implied when using `-l`, so you don't need it.

**Comment by ForbesLindesay** [comment]
I'm having a similar problem, it doesn't seem to be injecting the update script for me.

P.S. the -w is implied by the -l, so can be left out.

**Comment by ForbesLindesay** [comment]
Ah, Just worked out what it is, We write the the socket:

``` javascript
{name: 'fileModified', args:['dirPath\\index.html']}
```

but the actual path in the browser is:

```
localhost:3000/
```

**Comment by ForbesLindesay** [comment]
It's because of the feature of automatically serving index.html for directories

**Comment by karthikv** [comment]
Oh, good catch. I'll address this shortly.

**Comment by ForbesLindesay** [comment]
Just submitted a patch for it (didn't see your comment) but if you want to fix it differently that's fine.

**Comment by karthikv** [comment]
This has been fixed in version 0.1.1. Let me know if you're still having issues!
